proceeding/text/introduction.tex

```diff
@@ -10,3 +10,16 @@ \section{Introduction}
 This is what gamma-sky.net was created for.
 
 \end{itemize}
+
+
+        The field of high-energy (HE) and very-high-energy (VHE) astronomy is growing tremendously – while only
+        a decade ago we no more than a handful of sources in the GeV range, today we have thousands, including
+        hundreds within the TeV range. This advancement has been made possible due to our novel Cherenkov telescope instruments.
+        Such systems, both on the ground and in space as satellites, have a higher angular resolution than ever before.
```

Comment:
This sounds like the space satellites are Cherenkov telescopes.
They are not.
Fermi-LAT is a pair production telescope:
http://imagine.gsfc.nasa.gov/science/toolbox/gamma_detectors2.html

Maybe it's possible to separate out the space / HE / GeV part and the ground-based / VHE / TeV part into separate sentences? And for the space-telescope part, I'd definitely suggest you mention Fermi-LAT explicitly, since it's THE telescope that has produced the best dataset we have and the only one we use on the webpage.